src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,37 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+
+    static final int REQUIRED_AGE = 18;
+    static final int REQUIRED_SYMBOLS = 6;
```

Comment:
What is REQUIRED_SYMBOLS. Is it possible to understand quikly?
```suggestion
    private static final int MIN_AGE = 18;
    private static final int MIN_PASSWORD_LENGTH = 6;
```

Reply:
Замінив